GNUmakefile

```diff
@@ -6,6 +6,14 @@ GO_LDFLAGS := $(shell tools/bin/ldflags)
 GOFLAGS = -ldflags "$(GO_LDFLAGS)"
 GCFLAGS = -gcflags "$(GO_GCFLAGS)"
 
+# LOOP Plugin version defaults
+ifndef COSMOS_SHA
+override COSMOS_SHA = "f740e9ae54e79762991bdaf8ad6b50363261c056"
+endif
+ifndef STARKNET_SHA
+override STARKNET_SHA = "9a780650af4708e4bd9b75495feff2c5b4054e46"
+endif
```

Comment:
`?=` is not sufficient because we have chained called with hardcoded flags, causing empty string values to come through here rather than just unset.